config/eigendaflags/cli.go

```diff
@@ -49,6 +50,14 @@ func withEnvPrefix(envPrefix, s string) string {
 // CLIFlags ... used for EigenDA client configuration
 func CLIFlags(envPrefix, category string) []cli.Flag {
 	return []cli.Flag{
+		&cli.StringFlag{
+			Name:     BackendsToEnableFlagName,
+			Usage:    "Configures which set of eigenDA backends to enable (valid options are V1, V2, or V1andV2).",
+			EnvVars:  []string{withEnvPrefix(envPrefix, "BACKENDS_TO_ENABLE")},
+			Value:    "V1",
+			Category: category,
+			Required: false,
+		},
```

Comment:
I think a comma-separated list of backends would be nicer?
<img width="735" alt="image" src="https://github.com/user-attachments/assets/309039d9-4c0c-4538-85ab-f0e23f716966" />

Reply:
I didn't consider this option- it's much nicer. [8948665](https://github.com/Layr-Labs/eigenda-proxy/pull/348/commits/894866510148c1abcca15e78eeffa8fabf6651ac)